Main.java

```diff
@@ -0,0 +1,73 @@
+package com.company;
+
+import java.io.*;
+import java.util.StringTokenizer;
+
+public class Main {
+
+    public static void main(String[] args){
+
+        if(args.length!=2)

```

Comment:
```suggestion
        if(args.length!=2) {

```

Opening bracket should be in the same line

Reply:
Sure. I will keep that in mind.